src/app/shared/components/product/components/product-list/product-list.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, input } from '@angular/core';
+import { ProductItemComponent } from '../product-item/product-item.component';
+import { RouterLink } from '@angular/router';
+import { ProductListComponentVM } from './product-list.component.vm';
+
+/**
+ * ProductListComponent
+ *
+ * Type: Presentational
+ *
+ * Scope:
+ * - Responsible for rendering a list of products.
+ * - Delegates the rendering of individual products to `ProductItem` components.
```

Comment:
This is rather an out-of-scope thing, that it is not responsible for rendering a product-item